guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc

```diff
@@ -3,28 +3,18 @@
 
 Before installing {Project} in an IPv6 network, ensure that you meet the following requirements:
 
-ifndef::satellite[]
-* If you plan to provision hosts from {Project} or {SmartProxy}, you must install {Project} and {SmartProxies} on a system with `grub2` version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher.
-For other operating systems, copy the newest GRUB build to `/var/lib/tftpboot/grub2/grubx64.efi`.
-endif::[]
```

Comment:
This is different to the original PR. Is this by design?

Reply:
Apologies, I assumed this to be part of `master` changes but it's not. Restoring back in all 3 PRs.